code/modules/hydroponics/seed_datums.dm

```diff
@@ -282,7 +282,7 @@
 	seed_name = "apple"
 	display_name = "apple tree"
 	mutants = list("poisonapple","goldapple")
-	chems = list("nutriment" = list(1,10))
+	chems = list("applejuice" = list(1,10), "nutriment" = list(1,10))
```

Comment:
I would recommend removing the nutriment here, it will create two different flavors; one that tastes like apple and other that tastes like apple juice.

Reply:
That's fine.